app/services/search/location_builder.rb

```diff
@@ -71,4 +71,8 @@ def build_location_filter(location, radius)
   def initialize_nationwide_search
     @location = nil
   end
+
+  def map_location_name
```

Comment:
This method can be deleted now, n'est pas?

Reply:
Of course. Don't know how that slipped through!